Makefile

```diff
@@ -1,18 +1,20 @@
+.PHONY: l
+l: ## Lint Go source files
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest && golangci-lint run
+
 .PHONY: t
-t:
+t: ## Run unit tests
 	go test -race -count=1 ./...
 
 .PHONY: f
-f:
+f: ## Format code
 	go fmt ./...
 
-
 .PHONY: c
-c:
+c: ## Measure code coverage
```

Comment:
This command doesn't work for me

```
[20/11/23 1:46:47] ~/GitHub/ccache (lint) $ make c
go test -race -covermode=atomic ./... -coverprofile=cover.out && \
        go tool cover -func cover.out \
                | grep -vP '[89]\d\.\d%' | grep -v '100.0%' \
                || true
?       github.com/karlseguin/ccache/v3/assert  [no test files]
ok      github.com/karlseguin/ccache/v3 12.104s coverage: 94.6% of statements
grep: invalid option -- P
usage: grep [-abcdDEFGHhIiJLlMmnOopqRSsUVvwXxZz] [-A num] [-B num] [-C[num]]
        [-e pattern] [-f file] [--binary-files=value] [--color=when]
        [--context[=num]] [--directories=action] [--label] [--line-buffered]
        [--null] [pattern] [file ...]
go tool cover: signal: broken pipe
rm cover.out

```

Reply:
Ah right..GNU grep is different than BSD grep.

The goal of that was just to remove functions with high coverage from the output..since something with 90% coverage is good enough for me. But in the latest commit, I just took that filter out...not much harm in seeing those. (I left the 100% coverage filter on, since that one should work as-is on any OS).